Gemfile

```diff
@@ -0,0 +1,32 @@
+source "https://rubygems.org"
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+gem "rails", "~> 5.1.7"
+gem "sqlite3"
```

Comment:
please remove it
and add gem "mysql2", run bundle install and start server

Reply:
How to solve this error?
![image](https://user-images.githubusercontent.com/68435978/90113674-4a49e600-dd84-11ea-94fc-c529b5eca1e2.png)

Reply:
done